hack/dockerfile/install/rootlesskit.installer

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
 # When updating, also update rootlesskit commit in vendor.conf accordingly
```

Comment:
Let's update this to reference `vendor.mod`.

Reply:
done